### PR TITLE
Fix TODO comments

### LINKED ISSUE #185: Add testcases for getMentionsData
Link: https://github.com/Brightscout/mattermost-plugin-msteams-sync/blob/8fbc18bfd33e13678e1657551c49cb388df3bd75/server/message_hooks.go#L745

### LINKED ISSUE #215: Add testcases for handleAttachments
Link: https://github.com/mattermost/mattermost-plugin-msteams-sync/blob/2f4491c32b8fcbb0d185480f77faacb370d78ab0/server/handlers/attachments.go#L29

### LINKED ISSUE #222: Add testcases for connect API
Link: https://github.com/mattermost/mattermost-plugin-msteams-sync/blob/2f4491c32b8fcbb0d185480f77faacb370d78ab0/server/api.go#L279-L280

### PR DESCRIPTION
**Pull request by raghavaggarwal2308**
Ticket Links:
1. Fixes #215 
2. Fixes #222 
3. Fixes #185

### CODE REVIEW

File: server/message_hooks_test.go [hunk 1]
```diff
@@ -2003,3 +2003,177 @@ func TestGetChatIDForChannel(t *testing.T) {
...
+					DisplayName: "mock-name",
+				}, nil)
+			},
+			SetupStore: func(store *storemocks.Store) {
+				store.On("MattermostToTeamsUserID", testutils.GetID()).Return(testutils.GetUserID(), nil)
+			},
+			ExpectedMentionsCount: 1,
+		},
+		{
+			Name:            "GetMentionsData: mentioned all and a specific user in a channel",
```
Comment (ayusht2810) [inline server/message_hooks_test.go:2102]:
```suggestion
			Name:            "GetMentionsData: mentioned all and a specific user in a group chat",
```

**Review by ayusht2810 — COMMENTED** [review]
fix ci

**Review by manojmalik20 — APPROVED** [review]

File: server/handlers/attachments.go [hunk 2]
```diff
@@ -21,7 +21,8 @@ func (ah *ActivityHandler) handleDownloadFile(weburl string, client msteams.Clie
 	return data, nil
 }
 
-func (ah *ActivityHandler) handleAttachments(userID, channelID string, text string, msg *msteams.Message) (string, model.StringArray, string) {
+// TODO: Remove unused parameter
+func (ah *ActivityHandler) handleAttachments(_, channelID string, text string, msg *msteams.Message, chat *msteams.Chat) (string, model.StringArray, string) {
```
Comment (manojmalik20) [inline server/handlers/attachments.go:25]: Please remove this unused parameter and the TODO comment.

File: server/handlers/attachments.go [hunk 3]
```diff
@@ -21,9 +21,7 @@ func (ah *ActivityHandler) handleDownloadFile(weburl string, client msteams.Clie
 	return data, nil
 }
 
-// TODO: Add unit tests for this function
-// TODO: Remove unused parameter
-func (ah *ActivityHandler) handleAttachments(_, channelID string, text string, msg *msteams.Message, chat *msteams.Chat) (string, model.StringArray, string) {
+func (ah *ActivityHandler) handleAttachments(channelID string, text string, msg *msteams.Message, chat *msteams.Chat) (string, model.StringArray, string) {
```
Comment (manojmalik20) [inline server/handlers/attachments.go:24]:
```suggestion
func (ah *ActivityHandler) handleAttachments(channelID, text string, msg *msteams.Message, chat *msteams.Chat) (string, model.StringArray, string) {
```

**Review by manojmalik20 — APPROVED** [review]

**Pull request merged**